core/hosting/HostWithCoreClrHost/src/SampleHost.cpp

```diff
@@ -0,0 +1,324 @@
+#include <stdio.h>
```

Comment:
It would be nice to add a license header to make it more obvious that the code can be reused.  Something like 

```C++
// Licensed to the .NET Foundation under one or more agreements.
// The .NET Foundation licenses this file to you under the MIT license.
```

Reply:
Good idea. Added.